charts/stable/shinobi/values.yaml

```
@@ -0,0 +1,62 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: shinobisystems/shinobi
+  # -- image tag
+  # @default -- chart.appVersion
+  tag: dev
+  # -- image pull policy
+  pullPolicy: Always
+
+# -- environment variables. See more environment variables in the [shinobi documentation](https://shinobi.video/docs/).
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+  # PLUGIN_KEYS:
+  # SSL_ENABLED:
+
+# -- Configures service settings for the chart.
+# @default -- See values.yaml
+service:
+  main:
+    ports:
+      http:
+        port: 8080
+
+ingress:
+  # -- Enable and configure ingress settings for the chart under this key.
+  # @default -- See values.yaml
+  main:
+    enabled: false
+
+# -- Configure persistence settings for the chart under this key.
+# @default -- See values.yaml
+persistence:
+  config:
+    enabled: false
+    mountPath: /config
+  customautoload:
+    enabled: false
+    mountPath: /home/Shinobi/libs/customAutoLoad
+  database:
+    enabled: false
+    mountPath: /var/lib/mysql
+  videos:
+    enabled: false
+    mountPath: /home/Shinobi/videos
+  plugins:
+    enabled: false
+    mountPath: /home/Shinobi/plugins
+  shm:
+    enabled: true
+    type: emptyDir
+    medium: Memory
+    sizeLimit: 1Gi
```

```suggestion
    medium: Memory
```

This isn't supported in all Kubernetes versions, not sure what will happen if you specify a sizeLimit when it's not supported by Kubernetes, but therefore it may be better to just leave it out by default.

---

I got this from the frigate chart: https://github.com/k8s-at-home/charts/blob/9d8ef6eee6ab1829750419fc1af17d079f73462b/charts/stable/frigate/values.yaml#L70